packages/manager/modules/bm-server-components/src/network-interfaces/index.js

```diff
@@ -0,0 +1,30 @@
+import angular from 'angular';
+
+import '@ovh-ux/ui-kit';
+import 'angular-translate';
+import '@ovh-ux/ng-translate-async-loader';
+
+import component from './component';
+import service from './interfaces.service';
+import interfaceAttach from './attach/interfaces-attach.module';
+import interfaceDetach from './detach/interfaces-detach.module';
+import interfaceRename from './rename/interfaces-rename.module';
+
+const moduleName = 'ovhManagerBmServerComponentsNetworkInterfacesComponent';
+
+angular
+  .module(moduleName, [
+    interfaceAttach,
+    interfaceDetach,
+    interfaceRename,
+    'oui',
+    'ui.router',
+    'ovh-api-services',
```

Comment:
same https://github.com/ovh/manager/pull/10018#discussion_r1334512760

Reply:
yes, the controller uses `OvhApiVrack`